src/transform/src/reduction.rs

```diff
@@ -53,6 +53,16 @@ impl FoldConstants {
                 }
                 for aggregate in aggregates.iter_mut() {
                     aggregate.expr.reduce(&input_typ);
+                    if let (expr::AggregateFunc::Count, MirScalarExpr::Column(c)) =
+                        (&aggregate.func, &aggregate.expr)
+                    {
+                        // An aggregate that is a count of non-nullable data can be replaced by
+                        // count(true).
```

Comment:
I see the change to ```true``` in action with the randomized workload, but no tests are failing because of this. This points to a hole in functional testing that needs to be filled in.